org_onebusaway_iphone_Prefix.pch

```diff
@@ -13,11 +13,11 @@
     #import "OBACommonV1.h"
     #import "OBACommon.h"
 
-    #import "TestFlight.h"
     #import "GAI.h"
     #import "GAIFields.h"
     #import "GAITracker.h"
     #import "GAIDictionaryBuilder.h"
+    #define TFLog NSLog
```

Comment:
Looks like this and the next line were just for TestFlight per https://github.com/OneBusAway/onebusaway-iphone/commit/db10ce3330f6b1ce47b3d485ba9a6b14dce5ef32.